docs/sharing-your-dark-projects.md

```diff
@@ -0,0 +1,20 @@
+---
+id: sharing-dark
+title: Sharing Your Dark Projects
+sidebar_label: Sharing Your Dark Projects
+---
+
+Thanks for being part of our private beta! We're excited for you to share your Dark projects. The most important thing to us is getting your feedback. We appreciate hearing about bugs, requests, etc, in the Slack. 
+
+That said, we're happy for you to share publicly. The best way to share code for now is often via screenshot (ex: [https://twitter.com/janiczek/status/1220519157135003649](https://twitter.com/janiczek/status/1220519157135003649)) although we're working on better ways to copy/paste out.
+
+Here's a couple examples of users sharing about Dark: 
```

Comment:
Maybe:
"Here's some examples of users sharing their Dark experiences and projects:"